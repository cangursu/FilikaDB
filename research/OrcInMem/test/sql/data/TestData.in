0,col0,0
1,col1,0.1
2,col2,0.2
3,col3,0.3
4,col4,0.4
5,col5,0.5
6,col6,0.6
7,col7,0.7
8,col8,0.8
9,col9,0.9