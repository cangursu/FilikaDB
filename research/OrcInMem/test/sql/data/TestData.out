{"col1": 0, "col2": "col0", "col3": 0}
{"col1": 1, "col2": "col1", "col3": 0.1}
{"col1": 2, "col2": "col2", "col3": 0.2}
{"col1": 3, "col2": "col3", "col3": 0.3}
{"col1": 4, "col2": "col4", "col3": 0.4}
{"col1": 5, "col2": "col5", "col3": 0.5}
{"col1": 6, "col2": "col6", "col3": 0.6}
{"col1": 7, "col2": "col7", "col3": 0.7}
{"col1": 8, "col2": "col8", "col3": 0.8}
{"col1": 9, "col2": "col9", "col3": 0.9}